tools/deploy_local_auth_secret.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+import deployment_options
+import utils
+
+def main():
+    secret_name = 'assisted-installer-local-auth-key'
+    deploy_options = deployment_options.load_deployment_options()
+
+    # NO OP if we don't apply manifests as we don't want the secret included in the operator bundle
+    if not deploy_options.apply_manifest:
+        return
```

Comment:
I'm not sure if this is the correct choice.
We have a separate issue for making the operator deploy this secret on startup so I assume we wouldn't want this in the manifests, right?

What do you think @rwsu ?

Reply:
@djzager maybe you can answer this one?

Reply:
OLM does support managing [Secrets manifests in the "Operator Package"](https://docs.openshift.com/container-platform/4.6/operators/understanding/olm-packaging-format.html). So it might be okay to include it until this all gets wrapped up into a new controller loop.

My confidence rating in this suggestion is about 80% :sunglasses:

Reply:
But ... wouldn't that mean that every deployment would get the same secret key?

Reply:
Yes it would. Making me glad I didn't say 100% confidence.

Then not including it in the generated manifests is correct and the operator can create and deploy this on startup.